### ISSUE TITLE
Transactions: Get Tab

### ISSUE THREAD

**Issue by jonathan-irvin** [two-jelly-beans/jelly-fin#2]
A list of all transactions for the month

**Comment by ajroberts0417** [comment]
Minimum information for transaction info output to user: 
- Amount
- Date
- Payee

(Feature: Sorting)
Options to sort by date & amount seem like the minimum sorting options here.

**Comment by jonathan-irvin** [comment]
Agreed.  The default would be sort date ascending and amount descending.  While that may not be _exactly_ how it would look in a bank account, that's the most logical way when you're calculating a balance.

**Comment by jonathan-irvin** [comment]
This relates to #5

**Comment by jonathan-irvin** [comment]
@emmanuelvr I already have an index in the transactions collection on Firestore, will likely need to modify it based off of the discussions in #1

**Comment by jonathan-irvin** [comment]
This issue was moved to jelly-fin/jelly-fin-web#13
